Features/Lib/PlatformUI.js

```diff
@@ -0,0 +1,407 @@
+        if( element == null){
+            return false;
+        }
+        element = element.getDOMNode();
+
+        var visibility = ( element.offsetParent !== null );
+
+        return visibility;
+    }
+    /**
+     * Specific function used to open a content Tree path
+     *
+     * @param   {array}     path
+     * @param   {integer}   level
+     * @param   {string}    rootSelectorId
+     * @returns {Number|Boolean}
+     */
+    this.openTreePath = function (path, level, rootSelectorId){
+        var rootNode;
+        if (rootSelectorId === '')
+            rootNode = Y.one('.ez-platformui-app-body'); // FIXME
+        else
+            rootNode = Y.one('#' + rootSelectorId);
+
+        if (!rootNode)
+            return false;
+
+        var result = false;
+        if (level < path.length - 1){
+            var nodes = rootNode.all('.ez-tree-node');
+
+            nodes.some(function (elem){
+                leafNode = elem.one('.ez-tree-navigate');
+                if (leafNode.get('text') === path[level]){
+                    if (!elem.hasClass('is-tree-node-open')){
+                        this.clickElement(elem.one('.ez-tree-node-toggle'));
+                        result = -1;
+                    }
+
+                    if (elem.hasClass('is-tree-node-loading'))
+                result = -1;
+
+            var newRoot = elem.one('ul.ez-tree-level');
+            if (newRoot)
+                result = newRoot.generateID();
+
+            return true;
+                }
+            }, this);
+            return result;
+        }
+        else{
+            leafNode = rootNode.one('.ez-tree-navigate');
+            return this.clickElement(leafNode);
+        }
+    }
+
+    /**
+     * Transforms a file into javascript data and the drops it on top of an element
+     *
+     * @param {String} fileName     name of the file
+     * @param {String} fileType     MIME type of the file
+     * @param {String} cotentBase64 contents of file in base 64
+     * @param {String} selector     element selector
+     */
+    this.simulateDropEventWithFile = function( fileName, fileType, contentBase64, selector ){
+        jsFile = new Blob( [this.base64DecToArr( contentBase64 )], {type: fileType});
+        jsFile.name = fileName;
+        jsFile.lastModifiedDate = new Date();
+        jsFile.webkitRelativePath = "";
+
+        var filesArray = Array();
+        filesArray.push( jsFile );
+        var event = document.createEvent("HTMLEvents");
+        event.initEvent("drop", true, true);
+        event.dataTransfer = { files: filesArray };
+        window.document.getElementsByClassName(selector)[0].dispatchEvent(event);
+    }
+
+    /**
+     * Makes an element visible by changing its CSS style
+     *
+     * @param {String} selector     element selector
+     */
+    this.changeCssDisplay = function( selector ){
+        Y.StyleSheet('NewStyle').set( selector, { display: 'inline' });
```

Comment:
For the record, you don't to use a dynamically generated stylesheet for that.
`Y.one(selector).setStyle('display', 'inline');` would do exactly the same (in more reliable manner)